--- origsrc/gtypist-2.8.3/src/gtypist.c	2009-02-18 10:12:43.000000000 -0600
+++ src/gtypist-2.8.3/src/gtypist.c	2010-10-21 12:31:34.398834100 -0500
@@ -2091,7 +2091,7 @@ This program is released under the GNU G
 
   // Quick hack to get rid of the escape delays
 #ifdef __NCURSES_H 
-  ESCDELAY = 1;
+  set_escdelay(1);
 #endif
 
   /* set up colour pairs if possible */
--- origsrc/gtypist-2.8.3/src/script.c	2009-02-18 08:58:48.000000000 -0600
+++ src/gtypist-2.8.3/src/script.c	2010-10-21 02:19:51.036292400 -0500
@@ -23,7 +23,7 @@
 #ifdef HAVE_LIBCURSES
 #include <curses.h>
 #else
-#include <ncurses.h>
+#include <ncurses/curses.h>
 #endif
 #include "error.h"
 
