--- origsrc/gtypist-2.8.1/doc/Makefile.am	2008-05-16 11:33:17.000000000 -0500
+++ src/gtypist-2.8.1/doc/Makefile.am	2008-06-02 19:58:22.093750000 -0500
@@ -8,9 +8,9 @@
 html_docs = gtypist.html gtypist.cs.html gtypist.es.html
 
 .texi.html:
-	$(MAKEINFO) --html --no-split --no-headers -o gtypist.html gtypist.texi
-	$(MAKEINFO) --html --no-split --no-headers -o gtypist.cs.html gtypist.cs.texi
-	$(MAKEINFO) --html --no-split --no-headers -o gtypist.es.html gtypist.es.texi
+	$(MAKEINFO) --html --no-split --no-headers -o gtypist.html $(srcdir)/gtypist.texi
+	$(MAKEINFO) --html --no-split --no-headers -o gtypist.cs.html $(srcdir)/gtypist.cs.texi
+	$(MAKEINFO) --html --no-split --no-headers -o gtypist.es.html $(srcdir)/gtypist.es.texi
 
 all: info html
 
